rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -114,6 +145,17 @@ def fixture_accountant(
     if start_with_valid_premium:
         premium = Premium(credentials=rotki_premium_credentials, username=username)
 
+    # add accounting rules to the database
+    data_updater = RotkiDataUpdater(
+        msg_aggregator=function_scope_messages_aggregator,
+        user_db=database,
+    )
+    with open(latest_accounting_rules) as f:
+        data_updater.update_accounting_rules(
+            data=json.loads(f.read())['accounting_rules'],
+            version=999999,  # only for logs
```

Comment:
What do you mean only for logs here? So that if you see 999999 to know it's the test?

Reply:
The version number is used inside of `update_accounting_rules` only to display it in the logs and doesn't do any check. If we see this high number we can detect easily were it comes from